src/Drupal/V8/BasicCommands.php

```diff
@@ -112,24 +122,49 @@ public function make($opts = [
 
         $this->mergeComposerConfig();
 
-        $this->_exec("composer --no-progress --working-dir={$docroot} {$boolOptions}--prefer-{$opts['prefer']} update");
-        $this->linkSitesDefault();
-        $this->linkModules();
-        $this->linkThemes();
-        $this->linkLibraries();
+        $this->_exec("composer --no-progress --working-dir={$docroot} {$boolOptions} install");
+        $this->docrootSymlink('src/site', 'docroot/sites/default');
+        $this->docrootSymlink('src/modules', 'docroot/modules/custom');
+        $this->docrootSymlink('src/themes', 'docroot/themes/custom');
+        $this->docrootSymlink('docroot/vendor/bower-asset', 'docroot/libraries');
         if ($opts['frontend'] === true) {
+            $this->say('BUILDING FRONTEND');
             $this->downloadInterra();
             $this->installInterra();
             $this->buildInterra();
         }
-        $this->updateDrush();
+
+        if (!$this->checkDrushCompatibility()) {
+            $this->io()->warning('Your version of Drush is incompatible with DKAN2. Please upgrade to the latest Drush 9.x! If you are using the dkan-tools docker setup, try "dktl updatedrush".');
+        }
     }
 
-    private function updateDrush() {
-        $this->_exec("rm -rf /root/.composer/vendor");
-        $this->_exec("rm -rf /root/.composer/composer.lock");
-        $this->_exec("rm -rf /root/.composer/composer.json");
-        $this->_exec("composer global require drush/drush:9.5.2");
+    /**
+     * Update the version of Drush used in the container.
+     */
+    public function updatedrush() {
```

Comment:
@dafeder We need a -y flag here so we can run this command from our CI without human interaction.

Reply:
Fixed in latest commit: https://github.com/GetDKAN/dkan-tools/pull/131/commits/6ed96bdf942e51496fb8091bc07ed9b314988331